app/src/main/java/com/mealsmadeeasy/model/UserProfile.kt

```diff
@@ -1,8 +1,26 @@
 package com.mealsmadeeasy.model
 
+const val FEET_TO_METERS = 0.3048
+const val INCHES_TO_METERS = 0.0254
+const val POUNDS_TO_KILOGGRAMS = 0.453592
```

Comment:
Can we make these private?

Reply:
ANDREW. HOW COULD YOU NOT TELL ME I MISSPELLED KILOGRAMS? What else is code review for??